log4j-appender/src/main/java/org/apache/kafka/log4jappender/KafkaLog4jAppender.java

```diff
@@ -193,6 +204,14 @@ public String getKerb5ConfPath() {
         return kerb5ConfPath;
     }
 
+    public int getTimeout() {
+        return timeout;
+    }
+
+    public void setTimeout(int timeout) {
+        this.timeout = timeout;
+    }
```

Comment:
Aren't these publicly visible changes that require a KIP?

Reply:
In my opinion it is not that big change to have a KIP for that. Timeout is only introduced because of testing purposes (didn't wanted to wait the default 60 sec timeout)

Reply:
@ijuma Do we use KIPs for adding configs to `KafkaLog4jAppender` (`ignoreExceptions` and `timeout` here)?
@akatona84 Perhaps we should use a name that is consistent with the producer property that the `timeout` config sets?

Reply:
I'm renaming it to maxBlockMs